js/lib/beautify-css.js

```diff
@@ -98,6 +99,7 @@
 
         var indentSize = options.indent_size ? parseInt(options.indent_size, 10) : 4;
         var indentCharacter = options.indent_char || ' ';
+        var preserve_newlines = (options.preserve_newlines === undefined) ? true : options.preserve_newlines;
```

Comment:
This would actually cause it to default to `true` not `false` as stated on line 44.